inc/options/common/header.php

```diff
@@ -1,166 +1,399 @@
+					'id'   => 'general',
+					'text' => esc_html__( 'General', 'w3-total-cache' ),
+				),
+				array(
+					'id'   => 'page_cache',
+					'text' => esc_html__( 'Page Cache', 'w3-total-cache' ),
+				),
+				array(
+					'id'   => 'minify',
+					'text' => esc_html__( 'Minify', 'w3-total-cache' ),
+				),
+				array(
+					'id'   => 'system_opcache',
+					'text' => esc_html__( 'Opcode Cache', 'w3-total-cache' ),
+				),
+				array(
+					'id'   => 'database_cache',
+					'text' => esc_html__( 'Database Cache', 'w3-total-cache' ),
+				),
+				array(
+					'id'   => 'object_cache',
+					'text' => esc_html__( 'Object Cache', 'w3-total-cache' ),
+				),
+			);
+
+			if ( Util_Environment::is_w3tc_pro( $config ) ) {
+				$anchors[] = array(
+					'id'   => 'fragmentcache',
+					'text' => esc_html__( 'Fragment Cache', 'w3-total-cache' ),
+				);
+			}
+
+			$anchors = array_merge(
+				$anchors,
+				array(
+					array(
+						'id'   => 'browser_cache',
+						'text' => esc_html__( 'Browser Cache', 'w3-total-cache' ),
+					),
+					array(
+						'id'   => 'cdn',
+						'text' => wp_kses(
+							sprintf(
+								// translators: 1 opening HTML abbr tag, 2 closing HTML abbr tag.
+								__(
+									'%1$sCDN%2$s',
+									'w3-total-cache'
+								),
+								'<abbr title="' . esc_attr__( 'Content Delivery Network', 'w3-total-cache' ) . '">',
+								'</abbr>'
+							),
+							array(
+								'abbr' => array(
+									'title' => array(),
+								),
+							)
+						),
+					),
+					array(
+						'id'   => 'reverse_proxy',
+						'text' => esc_html__( 'Reverse Proxy', 'w3-total-cache' ),
+					),
+				)
+			);
+
+			if ( Util_Environment::is_w3tc_pro() ) {
+				$anchors[] = array(
+					'id'   => 'amazon_sns',
+					'text' => esc_html__( 'Message Bus', 'w3-total-cache' ),
+				);
+			}
+
+			$anchors[] = array(
+				'id'   => 'monitoring',
+				'text' => esc_html__( 'Monitoring', 'w3-total-cache' ),
+			);
+
+			if ( $licensing_visible ) {
+				array(
+					'id'   => 'licensing',
+					'text' => esc_html__( 'Licensing', 'w3-total-cache' ),
+				);
+			}
+
+			$link_attrs = array_merge(
+				$anchors,
+				$custom_areas,
+				array(
+					array(
+						'id'   => 'miscellaneous',
+						'text' => esc_html__( 'Miscellaneous', 'w3-total-cache' ),
+					),
+					array(
+						'id'   => 'debug',
+						'text' => esc_html__( 'Debug', 'w3-total-cache' ),
+					),
+					array(
+						'id'   => 'settings',
+						'text' => esc_html__( 'Import / Export Settings', 'w3-total-cache' ),
+					),
+				)
+			);
+
+			$links = array();
+			foreach ( $link_attrs as $link ) {
+				$links[] = "<a href=\"#{$link['id']}\">{$link['text']}</a>";
+			}
+
+			$links[] = '<a href="#" class="button-self-test">Compatibility Test</a>';
+
+			?>
+			<p id="w3tc-options-menu">
+				<?php
+				echo wp_kses(
+					implode( ' | ', $links ),
+					array(
+						'a'    => array(
+							'href'  => array(),
+							'class' => array(),
+						),
+						'abbr' => array(
```

Comment:
Are `abbr` elements used here?  I didn't see it.

Reply:
Removed abbr from wp_kses config as it isn't a part of the passed markup